content/accessibility/alternative-text-for-images.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Alternative Text for Images
+---
+
+## Overview
+
+> Text alternatives are a primary way for making information accessible because they can be rendered through any sensory modality (for example, visual, auditory or tactile) to match the needs of the user. Providing text alternatives allows the information to be rendered in a variety of ways by a variety of user agents. For example, a person who cannot see a picture can have the text alternative read aloud using synthesized speech.
+>
+> Source: https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html
```

Comment:
Do we need the `>` here? Can we just have this as regular text and not quoted text, or was there a reason for doing it this way?

Reply:
Well, this is a literal quote from https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html, that's why the quote block is used. Logically and semantically should be correct. Let me know if you disagree or you want me to rephrase the paragraph so it's not literal quoting anymore